Rubberduck.CodeAnalysis/Inspections/Concrete/ImplementedInterfaceMemberInspection.cs

```diff
@@ -40,49 +40,57 @@ namespace Rubberduck.CodeAnalysis.Inspections.Concrete
     /// ]]>
     /// </module>
     /// </example>
-    internal sealed class ImplementedInterfaceMemberInspection : DeclarationInspectionBase
+    internal sealed class ImplementedInterfaceMemberInspection : DeclarationInspectionBase<IEnumerable<ModuleBodyElementDeclaration>>
     {
         public ImplementedInterfaceMemberInspection(IDeclarationFinderProvider declarationFinderProvider)
-            : base(declarationFinderProvider, DeclarationType.ClassModule)
+            : base(declarationFinderProvider, DeclarationType.ClassModule) 
         {}
 
-        protected override bool IsResultDeclaration(Declaration declaration, DeclarationFinder finder)
+        protected override (bool, IEnumerable<ModuleBodyElementDeclaration>) IsResultDeclarationWithAdditionalProperties(Declaration declaration, DeclarationFinder finder)
         {
-            if (!IsInterfaceDeclaration(declaration))
+            if (!(declaration is ClassModuleDeclaration classModule && classModule.IsInterface)) 
             {
-                return false;
+                return (false, null);
             }
 
+            var implementedMembers = HasImplementedMembers(declaration, finder);
+            return (implementedMembers.Count() > 0, implementedMembers);
+        }
+
+        private IEnumerable<ModuleBodyElementDeclaration> HasImplementedMembers(Declaration declaration, DeclarationFinder finder) 
```

Comment:
This name suggests a `bool` return value; it should rather be names `ImplementedMembers`.